x-pack/plugins/synthetics/public/legacy_uptime/pages/overview.tsx

```diff
@@ -20,7 +20,7 @@ import { FilterGroup } from '../components/overview/filter_group/filter_group';
 
 const EuiFlexItemStyled = styled(EuiFlexItem)`
   && {
-    min-width: 800px;
+    min-width: 700px;
```

Comment:
Why this change?

Reply:
i felt filters were taking too much space. and search box needed more width